frontend/src/components/styled_list_items.tsx

```diff
@@ -0,0 +1,117 @@
+import { List, Typography } from "antd";
+import { DateTime } from "luxon";
+import { Link } from "react-router-dom";
+import { Activity } from "services/activities_service";
+import { Item } from "services/item_service";
+import { capitalizeWords } from "services/utils";
+import styled from "styled-components";
+
+export interface ItemListItemProps { 
+    item: Item
+    path: string
+    actions?: Array<any>
+    extras?: Array<any>
+}
+export interface ActivityListItemProps { 
+    item: Activity
+    path: string
+    actions?: Array<any>
+    extras?: Array<any>
+}
+
+const StyledListItem = styled(List.Item)`
+    &:hover {
+        background-color: rgba(200, 200, 235, .4);
+        transition: all .15s ease-in;
+    }
+`
+const StyledAvatar = styled.div`
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    gap: 10px;
+
+`
+const StyledDescription = styled.div`
+    display: flex,
+    flex-direction: row,
+    gap: 5px,
+    align-items: center,
+`
+
+export const ItemListItem = ({item, path, actions, extras}: ItemListItemProps) => {
+    return (
+        <Link to={{pathname: path}}>
+            <StyledListItem actions={actions?.map(i => i)} extra={extras?.map(i => i)}>
+                <List.Item.Meta 
+                    title={<div>{item.name}</div>}
+                    avatar={
+                        <StyledAvatar>
+                            {item.icon_url ? (
+                                <img
+                                    style={{ height: '50px', width: '50px' }}
+                                    src={item.icon_url}
+                                />
+                            ) : (
+                                <QuestionOutlined />
+                            )}
+                            {capitalizeWords(item.item_type)}
+                        </StyledAvatar>
+                    }
+                />
+            </StyledListItem>
+        </Link>)
+}
+
+export const ActivityListItem = ({item, path, actions, extras}: ActivityListItemProps) => {
+    return (
+        <Link to={{pathname: path}}>
+            <StyledListItem actions={actions?.map(i => i)} extra={extras?.map(i => i)}>
+                <List.Item.Meta 
+                title={item.item_name}
+                avatar={
+                    <StyledAvatar>
+                        {item.icon_url ? (
+                            <img
+                                style={{ height: '50px', width: '50px' }}
+                                src={item.icon_url}
+                            />
+                        ) : (
+                            <QuestionCircleOutlined />
+                        )}
+                        {capitalizeWords(item.item_type)}
+                    </StyledAvatar>
+                }
+                description={
+                    //These wrappers are not really necessary probably better to figure out best way to target the element that antd generates
```

Comment:
the _proper_ way is to adjust the theme here; the next proper way is to drop `& .ant-list-item-meta-description {whatever styles}` into the StyledListItem - some things may need !important to actually beat ants styles